WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -15,12 +15,42 @@ final class DashboardViewController: UIViewController {
     private let dashboardUIFactory: DashboardUIFactory
     private var dashboardUI: DashboardUI?
 
+    private var isDefaultSiteNameAvailable: Bool {
+        return ServiceLocator.stores.sessionManager.defaultSite?.name != nil && ServiceLocator.featureFlagService.isFeatureFlagEnabled(.largeTitles)
+    }
+
+    private var titleName: String {
+        if !isDefaultSiteNameAvailable {
+            return ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.title
+        }
+        return Localization.title
+    }
+
     // MARK: Subviews
 
     private lazy var containerView: UIView = {
         return UIView(frame: .zero)
     }()
 
+    private lazy var storeNameLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 1
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.subheadline
+        label.backgroundColor = .listForeground
+        label.textColor = UIColor.text
+        return label
+    }()
+
+    private lazy var stackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .listForeground
+        view.layoutMargins = UIEdgeInsets(top: 0, left: navigationController?.navigationBar.layoutMargins.left ?? 0, bottom: 0, right: 0)
```

Comment:
I think we need to use `directionalLayoutMargins` to properly support RTL lenguages.
```suggestion
        view.layoutMargins = UIEdgeInsets(top: 0, left: navigationController?.navigationBar.directionalLayoutMargins.leading ?? 0, bottom: 0, right: 0)
```

Reply:
I think you missed this comment! :-)

Reply:
Yes, I didn't realize about this comment. It's fixed!